common/src/main/java/org/apache/celeborn/common/network/server/memory/MemoryManager.java

```diff
@@ -38,69 +39,85 @@
 import org.apache.celeborn.common.util.ThreadUtils;
 import org.apache.celeborn.common.util.Utils;
 
-public class MemoryTracker {
-  private static final Logger logger = LoggerFactory.getLogger(MemoryTracker.class);
-  private static volatile MemoryTracker _INSTANCE = null;
+public class MemoryManager {
+  private static final Logger logger = LoggerFactory.getLogger(MemoryManager.class);
+  private static volatile MemoryManager _INSTANCE = null;
   private long maxDirectorMemory = 0;
   private final long pausePushDataThreshold;
   private final long pauseReplicateThreshold;
   private final long resumeThreshold;
   private final long maxSortMemory;
-  private final List<MemoryTrackerListener> memoryTrackerListeners = new ArrayList<>();
+  private final List<MemoryPressureListener> memoryPressureListeners = new ArrayList<>();
 
   private final ScheduledExecutorService checkService =
-      ThreadUtils.newDaemonSingleThreadScheduledExecutor("memory-tracker-check");
+      ThreadUtils.newDaemonSingleThreadScheduledExecutor("memory-manager-check");
```

Comment:
`memory-manager-checker`?

Reply:
Yes, it should because the memory tracker will have to manage the read buffer and memory stored shuffle data. So it is renamed now.

Reply:
I mean should use `checker`?

Reply:
OK, updated.